MODULE.bazel

```diff
@@ -13,9 +13,13 @@ git_override(
 bazel_dep(name = "rules_boost", repo_name = "com_github_nelhage_rules_boost")
 archive_override(
     module_name = "rules_boost",
-    integrity = "sha256-Sk6Rjh7r4pUyRYQ4QO2qhgZYvdkWbyrLJTCl9oY6Z7I=",
-    strip_prefix = "rules_boost-21cd44506bd15e06db89aa2866b9be90002d8eec",
-    urls = "https://github.com/nelhage/rules_boost/archive/21cd44506bd15e06db89aa2866b9be90002d8eec.tar.gz",
+    integrity = "sha256-11a92WLh9/4kJKqJMVFMXNseylj1Yo9Tm2p7j+aqZO4=",
+    patch_strip = 1,
+    patches = [
+        "//bazel/thirdparty:boost_iterator_fusion_pr_583.patch",
```

Comment:
This just merged, should we override the sha?

Reply:
yeh that was fast. fixed